app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <domain-config cleartextTrafficPermitted="true">
+        <domain includeSubdomains="true">monkeyuser.com</domain>
+    </domain-config>
+</network-security-config>
```

Comment:
I think this and the declaration need to be separated as a new PR, can't combine as this could relate to different configurations per different project
https://developer.android.com/training/articles/security-config

Reply:
@sleepylee PR is separated #39 🙏